src/addrdb.cpp

```diff
@@ -220,6 +220,10 @@ std::vector<CAddress> ReadAnchors(const fs::path& anchors_db_path)
         anchors.clear();
     }
 
-    fs::remove(anchors_db_path);
     return anchors;
 }
+
+void DeleteAnchorsFile(const fs::path& anchors_db_path)
+{
+    fs::remove(anchors_db_path);
+}
```

Comment:
Maybe inline this function definition into the declaration, as it is only one line of code (otherwise, missing EOF newline).

Reply:
Same reason of `delete_anchors_file`. I also prefer inline, but there are other functions in this file that is not following this pattern (e.g. `ReadFromStream`).

Reply:
ISTM that this is a per-function (performance and LOC/code organization) decision, not a style one.

Reply:
Great, going to change it.